### ISSUE TITLE
add time in random position

### ISSUE THREAD

**Issue by cll00** [sandydoo/flux-screensavers#23]
hi @sandydoo a nice feature may be a option to display time in a random place

**Comment by nerdpersonthing** [comment]
Agreed - an option to display the time is currently the only thing missing from this screensaver, in my opinion. Would be a very nice addition.

I would add that the position doesn't necessarily have to be random; there could be an option to configure where the time is positioned on the screen (potentially including a random option).
